Northstar.Custom/mod.json

```diff
@@ -416,13 +409,7 @@
 			}
 		}
 	],
-	"Maps": [
-		{
-			"Name": "mp_skyway_v1",
-			"VPK": "englishclient_mp_skyway_v1",
-			"RPak": "sp_skyway_v1"
-		}
```

Comment:
What's up with this being removed?

Reply:
because i put it there for testing like ages ago and it shouldn't be there

Reply:
like it literally does nothing, mod format doesn't support those keys

Reply:
lol aight